### DISCUSSION TITLE
Option to allow a page to load, but block images and video.

### DISCUSSION THREAD

**Discussion by ipfreely74** [proginosko/LeechBlockNG#489] [Ideas]
This is by far the best blocking extension. I like how it can take actions on individual pages instead of entire websites like most do. This extension would be perfect if there was an option to allow a page to load but block all images and video, when a block is triggered.  This would be ideal for sites that have mixed content. This option could be in the same section as the filters are. I know browsers have a built in feature to block images, but they only are able to block entire sites unconditionally. Please please please add this feature.

**Comment by proginosko** [comment]
This is closely related to #402. Good suggestion.
